servicebroker/test/models/binding_test.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+var expect = require('chai').expect;
+var serviceInstance = require('../../lib/models')().service_instance;
+var binding = require('../../lib/models')().binding;
+
+var fake_serviceInstanceId = "test_serviceinstance";
+var fake_orgId = "test_org";
+var fake_spaceId = "test_space";
+var fake_appId = "test_app";
+var fake_bindingId = "test_binding";
+
+var fake_appId2 = "test_app2";
+var fake_bindingId2 = "test_binding2";
+
+var fake_service_condition = {
+  'serviceInstanceId': fake_serviceInstanceId,
+  'orgId': fake_orgId,
+  'spaceId': fake_spaceId,
+  where: { 'serviceInstanceId': fake_serviceInstanceId, 'orgId': fake_orgId, 'spaceId': fake_spaceId },
+
+};
+var fake_binding_condition = {
+  'serviceInstanceId': fake_serviceInstanceId,
+  'appId': fake_appId,
+  'bindingId': fake_bindingId,
+  'timestamp': new Date().getTime() 
+};
+var where_condition = {where: { 'serviceInstanceId': fake_serviceInstanceId, 'appId': fake_appId, 'bindingId': fake_bindingId }};
+var fake_binding_condition2 = {
+  'serviceInstanceId': fake_serviceInstanceId,
+  'appId': fake_appId2,
+  'bindingId': fake_bindingId,
+  'timestamp': new Date().getTime() 
+};
+
+describe('Binding Model Definition Test Suite', function() {
+
+  before(function(done) {
+    serviceInstance.sequelize.sync().then(function(result) {
+      binding.sequelize.sync().then(function(result) {
```

Comment:
We need to remove sequelize.sync() since it recreates the tables and doesn't validate that your liquibase table definitions are correct.

The right approach is to just call sequelize.truncate() which will truncate all tables.